rlclientlib/live_model_impl.cc

```diff
@@ -83,12 +83,19 @@ int live_model_impl::choose_rank(
 
   // check arguments
   RETURN_IF_FAIL(check_null_or_empty(event_id, context, _trace_logger.get(), status));
-  if (!_model_ready)
-  {
-    RETURN_IF_FAIL(explore_only(event_id, context, response, status));
-    response.set_model_id("N/A");
-  }
-  else { RETURN_IF_FAIL(explore_exploit(event_id, context, response, status)); }
+
+  // The seed used is composed of uniform_hash(app_id) + uniform_hash(event_id)
+  const uint64_t seed = VW::uniform_hash(event_id, strlen(event_id), 0) + _seed_shift;
+
+  std::vector<int> action_ids;
```

Comment:
It's unfortunate that we don't know the size here so we can potentially avoid multiple allocations as this vector grows.